packages/example-project/component-library-angular/src/directives/proxies.ts

```diff
@@ -70,27 +55,28 @@ export class MyButton {
   }
 }
 
-import { CheckboxChangeEventDetail as ICheckboxCheckboxChangeEventDetail } from 'component-library';
-export declare interface MyCheckbox extends Components.MyCheckbox {}
+export declare interface MyButton extends Components.MyButton {
+  /**
+   * Emitted when the button has focus.
+   */
+  myFocus: EventEmitter<CustomEvent<void>>;
+  /**
+   * Emitted when the button loses focus.
+   */
+  myBlur: EventEmitter<CustomEvent<void>>;
+}
+
 @ProxyCmp({
-  tagName: 'my-checkbox',
-  customElement: MyCheckboxCmp,
   inputs: ['checked', 'color', 'disabled', 'indeterminate', 'mode', 'name', 'value'],
 })
 @Component({
   selector: 'my-checkbox',
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: '<ng-content></ng-content>',
+  // eslint-disable-next-line @angular-eslint/no-inputs-metadata-property
```

Comment:
Yes, this is fairly common for using non-standard, but public APIs that Angular supports. We intentionally use the `inputs` array instead of explicit `@Input` properties as Angular parses and generates the output slightly differently. 

This PR does not change the generated output of the Angular output target, but in the future it should follow what the other output targets do with disabling eslint for the entire file. It currently does it for tslint. With that change we could remove the inline disable rules completely.